android/src/reactNativeVersionPatch/RuntimeExecutor/latest/com/swmansion/reanimated/NativeProxy.java

```diff
@@ -23,6 +24,7 @@ public NativeProxy(ReactApplicationContext context, String valueUnpackerCode) {
         super(context);
         ReactFeatureFlagsWrapper.enableMountHooks();
         CallInvokerHolderImpl holder =
+                context.isBridgeless() ? null :
```

Comment:
can we somehow avoid passing null as argument?

Reply:
Done in [2efade7](https://github.com/software-mansion/react-native-reanimated/pull/5734/commits/2efade7e68c277b2ab8ca61c31814a910dce2703)